### PR TITLE
Fix a couple of linter problems, delete TODO.md

### PR DESCRIPTION
**Pull request by hughcapet**
- remark-lint
- Hadolint

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -43,18 +43,18 @@ RUN set -ex \
...
-    && ln -s ../../postgres0.yml $PGHOME/.config/patroni/patronictl.yaml \
+    && mkdir -p "$PGHOME/.config/patroni" /patroni /run/haproxy \
+    && ln -s ../../postgres0.yml "$PGHOME/.config/patroni/patronictl.yaml" \
     && ln -s /patronictl.py /usr/local/bin/patronictl \
     && sed -i "s|/var/lib/postgresql.*|$PGHOME:/bin/bash|" /etc/passwd \
     && chown -R postgres:postgres /var/log \
 \
     # Download etcd
-    && curl -sL https://github.com/coreos/etcd/releases/download/v${ETCDVERSION}/etcd-v${ETCDVERSION}-linux-$(dpkg --print-architecture).tar.gz \
+    && curl -sL https://github.com/coreos/etcd/releases/download/v${ETCDVERSION}/etcd-v${ETCDVERSION}-linux-"$(dpkg --print-architecture)".tar.gz \
```
Comment (CyberDem0n) [inline Dockerfile:53]:
```suggestion
    && curl -sL "https://github.com/coreos/etcd/releases/download/v$ETCDVERSION/etcd-v$ETCDVERSION-linux-$(dpkg --print-architecture).tar.gz" \
```

File: Dockerfile [hunk 2]
```diff
@@ -43,18 +43,18 @@ RUN set -ex \
...
             | tar xz -C /usr/local/bin --strip=1 --wildcards --no-anchored etcd etcdctl \
 \
     # Download confd
-    && curl -sL https://github.com/kelseyhightower/confd/releases/download/v${CONFDVERSION}/confd-${CONFDVERSION}-linux-$(dpkg --print-architecture) \
+    && curl -sL https://github.com/kelseyhightower/confd/releases/download/v${CONFDVERSION}/confd-${CONFDVERSION}-linux-"$(dpkg --print-architecture)" \
```
Comment (CyberDem0n) [inline Dockerfile:57]:
```suggestion
    && curl -sL "https://github.com/kelseyhightower/confd/releases/download/v$CONFDVERSION/confd-$CONFDVERSION-linux-$(dpkg --print-architecture)" \
```

**Review by CyberDem0n — COMMENTED** [review]
Since you started removing trailing white spaces lets do the same for all text files:
```
$ find . -name '*.md' -or -name '*.rst' | xargs grep -lE ' $'
./docs/README.rst
./docs/releases.rst
./docs/SETTINGS.rst
./docs/security.rst
./kubernetes/openshift-example/test/README.md
./kubernetes/openshift-example/README.md
./extras/startup-scripts/README.md
```

**Comment by CyberDem0n** [comment]
:+1:

**Comment by CyberDem0n** [comment]
:+1:

**Comment by hughcapet** [comment]
👍

**Pull request merged**